src/assets/locale/en/dataTables.json

```
@@ -0,0 +1,135 @@
+{
+  "title": "Data Tables Studio",
+  "description": "Generate structured tables from your chats, documents, and knowledge base using natural language prompts.",
```

Biome (2.1.2)

[error] The key description was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)